src/EditorFeatures/CSharp/BlockCommentEditing/BlockCommentEditingCommandHandler.cs

```diff
@@ -170,7 +171,41 @@ private static bool IsCaretInsideBlockCommentSyntax(SnapshotPoint caretPosition)
             var trivia = syntaxTree.FindTriviaAndAdjustForEndOfFile(caretPosition, CancellationToken.None);
 
             var isBlockComment = trivia.IsKind(SyntaxKind.MultiLineCommentTrivia) || trivia.IsKind(SyntaxKind.MultiLineDocumentationCommentTrivia);
-            return isBlockComment && trivia.FullSpan.Start < caretPosition;
+            if (isBlockComment)
+            {
+                var span = trivia.FullSpan;
+                if (span.Start < caretPosition && caretPosition < span.End)
```

Comment:
Congratulations, you found another bug! I tried `/**$$/` and it results `* $$/` on the new line, not expected! But the problem is not caused by this line here. And this issue is not what this PR trying to fix.

I'll look into that.